### ISSUE TITLE
M...bag cannot trigger rituals on metal runes but consumes items anyway

### ISSUE THREAD

**Issue by AranaiRa** [Mithion/Mana-And-Artifice#936]
**Describe the bug**
When triggering a metal ritual rune pattern with a Magnificent M...bag, the ritual does not fire. If you then check the M...bag's contents, the reagents have been consumed for despite no effect.

**To Reproduce**
1. Set up the Ritual of the Ancient Stone
2. Save it in a Magnificent M...bag
3. Make sure all reagents are in the bag
4. Right click the center rune
5. Check the bag's contents

**Version Information (please complete the following information):**
 - Mod Version: 3.0.0.24

**Comment by Mithion** [comment]
Just tried it, and it worked fine.  What ritual, tier, faction?

**Comment by Sarithule** [comment]
I have just had this happen to me with the practitioners pouch
Ritual - Ritual of the Ancient Stone
Tier - I am Tier 3
Faction - I am of the Wizards Council

To Reproduce

    1. Set up the Ritual of the Ancient Stone with metal runes, use regular sand, and use the empty hand auto fill function to set up the ritual.
    2. Store the ritual in the Practitioners pouch
    3. Complete the ritual to receive a Mote of Earth 
    4. Right click the center rune with the Practitioners pouch (it will appear as if nothing happens)
    5. Check the bag's contents and see that the ritual components have been withdrawn while the ritual was not initiated
 
![2024-09-08_23 25 53](https://github.com/user-attachments/assets/36fcb882-b7cb-442d-a75c-8a3c87436997)


Version Information (please complete the following information):

    Mod Version: 3.0.0.24
Forge Version: 47.3.0

**Comment by Sarithule** [comment]
Ok, even more weirdness with the Ritual of the Ancient Stone:

Since the Practitioners Pouch was not was not working with the metal runes with the Ritual of the Ancient Stone, I decided to do it with chalk runes,

I put some chalk in the Practitioners Pouch,
Picked up the metal runes and placed chalk runes down.
I created the ritual and then stored it in the practitioners pouch.
I completed the ritual to obtain a Mote of Earth.
I used the practitioners pouch to start another Ritual of the Ancient Stone which appeared to work correctly and produced a Mote of Earth when finished.
I opened the Practitioners pouch to find that the chalk had not been used at all.
I repeated the ritual a few times to see if it would use any chalk and it did not.
I took all the chalk out of the Practitioners Pouch and tried to start the ritual with the Practitioners Pouch.
The ritual worked even without any chalk in the Practitioners Pouch.

So when the Ritual of Ancient Stone is created with chalk and stored in the Practitioners pouch, the Practitioners Pouch does not Require chalk to be in it nor does it use chalk if it is present when producing the Ritual of the Ancient Stone.

Mod Version: 3.0.0.24
Forge Version: 47.3.0
